app/scripts/events.js

```diff
@@ -186,13 +188,19 @@ define([
                 }
             });
 
-            $('.em-wrap').on('click', function() {
-                var code = $(this).find('.em').data('unicode'),
-                    $curTextarea = $('.l-chat:visible .textarea'),
-                    val = $curTextarea.html();
+            $workspace.on('click', '.j-em', function(event) {
+                Cursor.setCursorAfterElement(event.target);
+            });
+
+            $('.smiles-group').on('click', function(event) {
+                var target = event.target,
```

Comment:
$(this) is a target if you use jQuery,
so don't mix jQuery and Native.

Use $(this) or better use Native 
`` addEventListener``